lib/features/composer/presentation/composer_controller.dart

```diff
@@ -1891,7 +1934,9 @@ class ComposerController extends BaseController with DragDropFileMixin implement
     if (identitySelected.value == null) {
       _getAllIdentities();
     } else {
-      await _selectIdentity(identitySelected.value);
+      if (composerArguments.value?.emailActionType != EmailActionType.editDraft) {
```

Comment:
why not base on `SignatureStatus`?

Reply:
Checking `SignatureStatus` won't work because that value is already changed at `_initIdentities`